### PR TITLE
pipes.sh and pipesX.sh

### PR DESCRIPTION
**Pull request by superloach**
Adds the pipes.sh terminal screensaver.

### CODE REVIEW

**Comment by JL2210** [comment]
Just a quick thought, I don't think that VLC will use this over the included one. Nice work, though!

**Comment by superloach** [comment]
Haha, I'm just making some easy packages I've been thinking about for ages.

**Comment by JL2210** [comment]
Lol. I did that once. Five random PRs in a day. I think they were `wireless_tools`, `firejail`, `hplip`, `checkinstall`, and `quakespasm`.

File: packages/pipes_sh.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  description 'Animated pipes terminal screensaver'
+  homepage 'https://pipeseroni.github.io/'
+  version '1.3.0'
+  source_url 'https://github.com/pipeseroni/pipes.sh/archive/v1.3.0.tar.gz'
+  source_sha256 '532976dd8dc2d98330c45a8bcb6d7dc19e0b0e30bba8872dcce352361655a426'
+
+  depends_on 'ncurses'
+
+  def self.install
+    system "make", "PREFIX=#{CREW_DEST_DIR}#{CREW_PREFIX}", "install"
```
Comment (uberhacker) [inline packages/pipes_sh.rb]: You can use `#{CREW_DEST_PREFIX}` instead of the two combined constants.

File: packages/pipesx_sh.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  description 'Animated pipes terminal screensaver at an angle'
+  homepage 'https://pipeseroni.github.io/'
+  version '1.1.0'
+  source_url 'https://github.com/pipeseroni/pipesX.sh/archive/v1.1.0.tar.gz'
+  source_sha256 '82e60efbe3e8ae1c83a9b7e5df7d67ccdad3905dfd478f27d2b2be488603420c'
+
+  depends_on 'ncurses'
+
+  def self.install
+    system "make", "PREFIX=#{CREW_DEST_DIR}#{CREW_PREFIX}", "install"
```
Comment (uberhacker) [inline packages/pipesx_sh.rb]: Same here.  Only `#{CREW_DEST_PREFIX}` is needed.

File: packages/pipes_sh.rb (hunk 1 above)
Comment (uberhacker) [reply packages/pipes_sh.rb]: With that said, it might be better to use `DESTDIR=#{CREW_DEST_DIR}` and the normal `PREFIX=#{CREW_PREFIX}`, if possible.

**Comment by uberhacker** [comment]
Great, just what we need; more screensavers to lock up our screen randomly when we need to start working. LoL

**Comment by JL2210** [comment]
Awesome!

**Comment by JL2210** [comment]
You know, that's one of the best parts about Windows. :rofl:

File: packages/pipes_sh.rb (hunk 1 above)
Comment (superloach) [reply packages/pipes_sh.rb]: So `make DESTDIR=#{CREW_DEST_DIR} PREFIX=#{CREW_PREFIX} install` for both packages?

**Comment by superloach** [comment]
Haha, yup. Also, this runs in a terminal, only when you run it. No unexpected lock-ups there!

File: packages/pipes_sh.rb (hunk 1 above)
Comment (uberhacker) [reply packages/pipes_sh.rb]: If you can swing it, sure.

**Comment by superloach** [comment]
Heck yeah >:3

**Pull request merged**